### PR TITLE
Web rtc porting

### PR DESCRIPTION
**Pull request by d-telnyx**

### CODE REVIEW

File: tests/api_resources/PortingOrderTest.php [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    }
+
+    public function testIsCreatable()
+    {
+        $this->expectsRequest(
+            'post',
+            '/v2/porting_orders'
+        );
+        $resource = PortingOrder::create(["phone_numbers" => ["+13035550000","+13035550001","+13035550002"]]);
+        $this->assertInstanceOf(\Telnyx\PortingOrder::class, $resource);
```
Comment (d-telnyx) [inline tests/api_resources/PortingOrderTest.php:31]: @weroh the request is a list of phone numbers and the response payload is a list of results. In this case, you should assert that the first object in the list is a `\Telnyx\PortingOrder`

File: lib/TelephonyCredential.php [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+namespace Telnyx;
+
+/**
+ * Class TelephonyCredential
+ *
+ * @package Telnyx
+ */
+class TelephonyCredential extends ApiResource
+{
+    const OBJECT_NAME = "telephony_credential";
```
Comment (d-telnyx) [inline lib/TelephonyCredential.php:12]: @weroh in this case, the endpoint is `telephony_credentials` however the `record_type` is `credential` 

```json
{
  "data": {
    "created_at": "2020-06-18T21:32:38",
    "expired": "false",
    "expires_at": "2042-06-18T21:32:38",
    "id": "c215ade3-0d39-418e-94be-c5f780760199",
    "name": "2020-06-18 21:32:38.917732Z",
    "record_type": "credential",
    "resource_id": "connection:804252963366242252",
    "sip_password": "a92dbcfb60184a8cb330b0acb2f7617b",
    "sip_username": "gencrednCvHU5IYpSBPPsXI2iQsDX",
    "updated_at": "2020-06-18T21:32:38.000Z",
    "user_id": "user-id"
  }
}
```

If possible, we'd like to keep the object name as `TelephonyCredential` to match the endpoint.

The function [convertToTelnyxObject](https://github.com/team-telnyx/telnyx-php/blob/master/lib/Util/Util.php#L43) is unable to match the `record_type` to this object as-is.

This is what is causing the credential tests to fail.

File: tests/api_resources/PortingOrderTest.php [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        $resource = PortingOrder::retrieve(self::TEST_RESOURCE_ID);
+        $this->expectsRequest(
+            'delete',
+            '/v2/porting_orders/' . urlencode(self::TEST_RESOURCE_ID)
+        );
+        $resource->delete();
+        $this->assertInstanceOf(\Telnyx\PortingOrder::class, $resource);
+    }
+
+    public function testLoaTemplate()
```
Comment (d-telnyx) [inline tests/api_resources/PortingOrderTest.php:65]: @weroh we can skip this test until we're out of beta

**Review by d-telnyx — COMMENTED** [review]
@weroh I took a quick look at what was causing tests to fail with notes. Please see review

File: tests/api_resources/PortingOrderTest.php (hunk 1 above)
Comment (weroh) [reply tests/api_resources/PortingOrderTest.php]: This is now expecting a collection and checking if the first object is a \Telnyx\PortingOrder

File: lib/TelephonyCredential.php (hunk 2 above)
Comment (weroh) [reply lib/TelephonyCredential.php]: The new version expects `record_type` to be credential while using `telephony_credentials` as the url

File: tests/api_resources/PortingOrderTest.php (hunk 3 above)
Comment (weroh) [reply tests/api_resources/PortingOrderTest.php:68]: Test has been commented out with a note

**Review by drewkitch — APPROVED** [review]

**Pull request merged**